tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -326,29 +349,33 @@ Polymer({
         ]);
   },
 
+  selectAll: function(query: string) {
+    return d3.selectAll(
+      Polymer.dom(this.root).querySelectorAll(query) as any);
+  },
+
   haveSaliency: function() {
-    if (!this.featuresList || !this.saliency ||
+    if (!this.filteredFeaturesList || !this.saliency ||
         Object.keys(this.saliency).length === 0 || !this.colors) {
       return;
     }
 
     // TODO(jwexler): Find a way to do this without requestAnimationFrame.
-    // If the paper-inputs for the features have yet to be rendered, wait to
-    // perform this processing. There should be paper-inputs for all non-image
+    // If the inputs for the features have yet to be rendered, wait to
+    // perform this processing. There should be inputs for all non-image
     // features.
-    if (d3.selectAll('.value input').size() <
-        (this.featuresList.length - Object.keys(this.imageInfo).length)) {
+    if (this.selectAll('input.value-pill').size() <
+        (this.filteredFeaturesList.length - Object.keys(this.imageInfo).length)) {
       requestAnimationFrame(() => this.haveSaliency());
       return;
     }
 
     // Reset all text to black
-    d3.selectAll<HTMLInputElement, {}>('.value-pill')
-        .style('background', 'lightgrey');
-
+    this.selectAll('.value-pill')
+        .style('background', '#e8eaed');
```

Comment:
I am not sure how this code relates to the comment in L373. Are they related? Also, would it be possible to hoist this constant color value as a module level constant?

Reply:
fixed comment and changed to use neutralSaliencyColor as it should